### ISSUE TITLE
Accessibility issue on user's management page

### ISSUE THREAD

**Issue by CoBC** [nextcloud/server#20768]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Is your feature request related to a problem? Please describe.**
Actual lists using to choose user's group or user's quota on users' management page are very dificult to use with a screenreader

**Describe the solution you'd like**
Replace all lists by classic html elements, for group use a select with attribute multiple, and for quota just use a select

**Describe alternatives you've considered**
No alternative

**Additional context**
No more to add

**Comment by kesselb** [comment]
cc @jancborchardt

**Comment by jancborchardt** [comment]
@Oreonan thanks a lot for the report and sorry for the issue! Since we are already in the latest beta and now release candidate phase of Nextcloud 19, we could only have this for Nextcloud 20 though.

@GretaD @skjnldsv does the user management already use the Vue components, or does this have to be solved manually? (And would a backport to Nextcloud 19 be possible at all?)

**Comment by skjnldsv** [comment]
Components

**Comment by rullzer** [comment]
>  (And would a backport to Nextcloud 19 be possible at all?)

kind of depends how big the changes are

**Comment by jancborchardt** [comment]
@GretaD can you look into this again as you now know your way around the user management?

**Comment by FriendlyGamer** [comment]
Hi is this in regards to the concerns I am having in the forum? https://help.nextcloud.com/t/accessibly-concern-cannot-set-quota-due-to-zoomed-in-browser-vivaldi/80036  

Attached is what I am having troubles with in regards to accessing certain components ( cutting off in the GROUP area and not being able to access others at all like QUOTAS). 
![Screenshot from 2020-05-05 11-05-30](https://user-images.githubusercontent.com/38051855/81094714-ecc71300-8ed1-11ea-9491-bf8227cbb810.png) 

Please let me know and thanks in advance!

**Comment by jancborchardt** [comment]
@FriendlyGamer that is a different problem – can I ask you to open a separate issue about this? Thank you :)

**Comment by Oreonan** [comment]
What about this issue? Still not fixed in NextCloud 23 :(

**Comment by GretaD** [comment]
Hello @Oreonan 

Im really sorry you havent had feedback from us since a while. I will poke my server colleagues or have a look soon myself.

**Comment by CoBC** [comment]
Still not fixed on NextCloud 25! What about this?

**Comment by skjnldsv** [comment]
Some have been done some are still left.
Follow the multiselect migration here: https://github.com/nextcloud/nextcloud-vue/pull/3435

**Comment by szaimen** [comment]
Hi, please update to 24.0.8 or better 25.0.2 and report back if it fixes the issue. Thank you!

**Comment by susnux** [comment]
The user management was fully migrated, like all other places in server core, to NcSelect instead of NcMultiselect with Nextcloud 28. So I think this is resolved now.
